openquake/hazardlib/geo/surface/multi_old.py

```diff
@@ -0,0 +1,675 @@
+# -*- coding: utf-8 -*-
```

Comment:
Was this file committed by mistake?

Reply:
No I simply want to also keep the old methodology for the time being.